movie-app/src/App.js

```diff
@@ -1,26 +1,24 @@
 import "./scss/main.scss";
 import React, { useState, useEffect } from "react";
-import { getMovieByName } from "./utilities/utils";
+import { getMoviesByName } from "./utilities/utils";
 import NavBar from "./header/NavBar";
 import Searched from './body/Searched';
+import { Home } from './body/Home';
 
 
 function App() {
   const [movie, setMovie] = useState({});
-  const [movieSearch, setMovieSearch] = useState("batman");
+  const [movieSearch, setMovieSearch] = useState();
 
   useEffect(() => {
-    getMovieByName(movieSearch).then((res) => setMovie(res));
+    getMoviesByName(movieSearch).then((res) => setMovie(res));
   }, [movieSearch])
 
-  
 
   return (
     <div className="App">
       <NavBar movieValue={movieSearch} setMovieSearch={setMovieSearch} />
-
-      {movie.Search ?  <Searched movies={movie.Search} />  :  <h2> Please enter a valid movie title </h2> }
-      
+      {movieSearch === undefined ?  <Home/>  :  (movie.Search ?  <Searched movies={movie.Search} />  :  <h2> Please enter a valid movie title </h2>) }
```

Comment:
You can invert the logic here a bit to simplify things.  This way, it'll also cover `null`.

```
{movieSearch ? (movie.Search ? <Searched movies={movie.Search} /> : <h2> Please enter a valid movie title </h2>) : <Home/>}
```